Ambiguous-Squids/bangazon-api
# Mm product auth

## Description
Temporary disable of session authentication to allow testing of product click-through when authenticated.
Change to views.py to help with authentication.

 ## Steps to Test
1. Fetch this branch AND `mm-product-click-through` branch from `Bangular` repo.
2. `python manage.py runserver` on this branch.
3. run `http-server` on the `Bangular` branch.
4. Navigate to localhost:8080/login
5. Login with the temp hard coded credentials `admin` `password1234`
6. Click `Products` in navbar.
7. confirm that you can click through all the way to product detail.
8. Open incognito browser.
9. Try to navigate to `http://localhost:8080/#/product_list/Music/3`.  This should tell you to log in.
10. Try to navigate to `http://localhost:8080/#/product_detail/5`.  This should tell you to log in.
11. Try to navigate to `http://localhost:8080/#/product_types`. This should tell you to log in.
